databases/week2/databases-week2-Homework/more_queries.sql

```diff
@@ -0,0 +1,50 @@
+-- Get all tasks assigned to users whose email ends in @spotify.com
+SELECT t.title
+FROM user_task ut 
+INNER JOIN task t ON ut.task_id=t.id 
+INNER JOIN (
+    SELECT u.id, u.email
+    FROM user u
+    WHERE email LIKE '%@spotify.com'
+) u ON ut.user_id=u.id;
+
+
+-- Get all taks for 'Donald Duck' where status 'not started' 
+SELECT t.title
+FROM task t
+INNER JOIN user_task ut ON t.id = ut.task_id
+INNER JOIN status s ON t.status_id = s.id 
+WHERE ut.user_id = (
+    SELECT id 
+    FROM user
+    WHERE name = 'Donald Duck'
+)
+AND s.id = (
+    SELECT id
+    FROM status
+    WHERE name = 'Not started'
+);
+
```

Comment:
They query fails if there are more than two whose names are Donald Duck.
Can you come up with a solution to this?

Reply:
I'm thinking it's the equal sign (=), so I changed it to 'LIKE' instead.